src/lua/thread.rs

```diff
@@ -130,7 +130,13 @@ pub fn init() {
     let (use_config, maybe_init_file) = init_path::get_config();
     if use_config {
         match maybe_init_file {
-            Ok(init_file) => {
+            Ok((init_dir, init_file)) => {
+                if init_dir.components().next().is_some() {
+                    // Add the config directory to the package path.
+                    let mut package: hlua::LuaTable<_> = lua.get("package").unwrap();
+                    let paths: String = package.get("path").unwrap();
```

Comment:
The commit I wanted to link is this 1bf3521a520b73905a4b2d5b8adc53f42ae407a4, but I find that overly verbose. `expect` seems like the best choice.